### PR TITLE
Stage 1 and 2 complete code

### PR DESCRIPTION
**Pull request by sm86**
Changed latest code to root folder and properties into a different file as per suggestions. 
Added the code for Phase 1: Leaf Node identification and Phase 2A: Algo for prominent node discovery.

### CODE REVIEW

**Review by jimkont — COMMENTED** [review]
Thanks! Can you delete the .classfiles and the eclipse related files? Then it's good to go :)

**Comment by sm86** [comment]
@jimkont Removed the files. thank you!

**Comment by jimkont** [comment]
Thanks @sm86 
some class files are still in the pull request, can you check again?
you are free to create folders for structuring the code but `root/DBTaxProject` it implied by the repo itself, and can, imo, be skipped

**Comment by jimkont** [comment]
[note that you do not need to fix all codacy issues with one go ;) ]

File: root/DBTaxProject/.classpath [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jimkont) [inline root/DBTaxProject/.classpath]: we can delete this one as well

Comment (sm86) [reply root/DBTaxProject/.classpath]: Deleted! Thanks

File: root/DBTaxProject/src/main/java/org/dbpedia/dbtax/database/EdgeDB.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		
+		//Connect to database
+		Connection connection = DatabaseConnection.getConnection();
+		
+		PreparedStatement ps = null;
+		String query =  "SELECT parent_id FROM edges WHERE child_id =?";
+		
+		ResultSet rs = null;
+		
+		try{
```
Comment (jimkont) [inline root/DBTaxProject/src/main/java/org/dbpedia/dbtax/database/EdgeDB.java]: it is easier to use the try with resources feature introduced in Java 7+
http://tutorials.jenkov.com/java-exception-handling/try-with-resources.html

**Comment by jimkont** [comment]
Hey @marfox,

I  see that we have copied code from http://www.mpi-inf.mpg.de/yago-naga/javatools
do you remember if we just copied the code or adjusted it in any way? there could be license incompatibility issues that we should check

File: root/DBTaxProject/src/main/java/org/dbpedia/dbtax/database/EdgeDB.java (hunk 2 above)
Comment (sm86) [reply root/DBTaxProject/src/main/java/org/dbpedia/dbtax/database/EdgeDB.java]: sounds good and simple! Will slowly transition to that.

**Comment by marfox** [comment]
If I remember well, we just reused yago code.

Il 11 giu 2017 8:57 AM, "Dimitris Kontokostas" <notifications@github.com>
ha scritto:

**Comment by jimkont** [comment]
Cool! then we could just reference it as a maven dependency as described [here](https://stackoverflow.com/questions/20161602/loading-maven-dependencies-from-github)
made an issue for this (https://github.com/dbpedia/DBTax/issues/6) so that we remember it

**Pull request merged**